monkey/agent_plugins/build_plugin.sh

```diff
@@ -35,6 +35,30 @@ rm requirements.txt
 pushd "$PLUGIN_PATH/src" || fail "$PLUGIN_PATH/src does not exist"
 
 source_archive=$PLUGIN_PATH/$SOURCE_FILENAME
+
+plugin_name=$(get_plugin_name "$PLUGIN_PATH")
+
+plugin_name_lowercase=$(lower "$plugin_name")
+plugin_options_filename="${plugin_name_lowercase}_options"
```

Comment:
Note: We're introducing a convention here, so we'll need to remember to document it